thermal.snbt

```diff
@@ -0,0 +1,1201 @@
+				}
+				title: "Any #thermal:crafting/dies"
+				type: "item"
+			}]
+			title: "Die!"
+			x: 2.0d
+			y: 3.0d
+		}
+		{
+			dependencies: ["0A13D9AFD4429192"]
+			description: [
+				"Using the sawdust from the sawmill, we can fashion a <rainb>Carboard Box!&r "
+				"You can use it on any block(unless the block is blacklisted) to preserve the data of the block as you transport it. "
+				"You can easily move spawners this way!"
+				""
+				"&eTip:&r"
+				"Right click on a block to wrap it with the Cardboard Box. Then, you can break it and transport it anywhere you like. Once you placed it back down, just shift+right click to remove the Cardboard Box. "
+				""
+				"&6Hint:&r"
+				"Think of all the breakable things you could preserve!"
+			]
+			id: "4CCA86CDD3745003"
+			subtitle: "Craft the <rainb>Legendary Cardboard Box!&r"
+			tasks: [{
+				id: "29F39634995255B9"
+				item: "mekanism:cardboard_box"
+				type: "item"
+			}]
+			title: "Moving Day!"
+			x: 3.0d
+			y: 3.0d
+		}
+		{
+			dependencies: [
+				"72B4BED27384E92F"
+				"7CAA549311FC1474"
+			]
+			description: ["Melt with the Magma Crucible, and bottle with the Fluid Encapsulator!"]
+			id: "6514AD78D683A071"
+			subtitle: "Liquids requried for &eThermal Series Alloys!&r"
+			tasks: [
+				{
+					id: "63689109448F47BA"
+					item: "thermal:redstone_bucket"
+					type: "item"
+				}
+				{
+					id: "37A9AE53E98B602D"
+					item: "thermal:glowstone_bucket"
+					type: "item"
+				}
+				{
+					id: "294FC85F4C209AFE"
+					item: "thermal:ender_bucket"
+					type: "item"
+				}
+			]
+			title: "Liquids for Alloying"
+			x: 5.0d
+			y: 4.0d
+		}
+		{
+			dependencies: ["5DA4E7AD6DB95A62"]
+			id: "791C5B1B1D6B17E5"
+			subtitle: "Brrrr"
+			tasks: [
+				{
+					count: 16L
+					id: "44F7D4F1F2B6750A"
+					item: { Count: 16, id: "minecraft:snowball" }
+					type: "item"
+				}
+				{
+					count: 16L
+					id: "443E0EB8FF05B8FA"
+					item: { Count: 16, id: "minecraft:ice" }
+					type: "item"
+				}
+			]
+			title: "Quite Chilly, Innit?"
+			x: 5.0d
+			y: 3.0d
+		}
+		{
+			dependencies: ["19D5DA2F39486B28"]
+			id: "2D140246BEB6A4C2"
+			subtitle: "Obtain &eBitumen&r from the Centrifugal Separator"
+			tasks: [{
+				count: 16L
+				id: "0BDAE3AEDEA5D111"
+				item: { Count: 16, id: "thermal:bitumen" }
+				type: "item"
+			}]
+			x: 7.0d
+			y: 3.0d
+		}
+		{
+			dependencies: ["20442358C1AFD7B8"]
+			description: ["The Satchel is a simple way to store your items!"]
+			id: "45D828E2ECBE4418"
+			subtitle: "Craft a &eSatchel&r"
+			tasks: [{
+				id: "37B0399233ADEC7A"
+				item: {
+					Count: 1
+					id: "thermal:satchel"
+					tag: { }
+				}
+				type: "item"
+			}]
+			x: -5.0d
+			y: -1.0d
+		}
+		{
+			dependencies: ["20442358C1AFD7B8"]
+			description: [
+				"Safely detonate any explosive from a distance! Onlookers will never know it was you!"
+				""
+				""
+				"&o&5AdamCo™ is not responsible for any home damages caused by detonations of explosives...&r"

```

Comment:
i think this should be more consistent with how it's been written, or atleast it's the same way i have, idk about adam or anyone else.
```suggestion
				"&o&5Adam Co. is not responsible for any home damages caused by detonations of explosives...&r"
```